data/json/items/armor/jewelry.json

```diff
@@ -253,8 +248,7 @@
     "material": [ "silver" ],
     "symbol": "[",
     "looks_like": "tieclip",
-    "color": "light_gray",
-    "flags": [ "FANCY" ]
+    "color": "light_gray"
```

Comment:
How often do you see people wearing these outside of formal occasions or by those that just like wearing fancy stuff?

Reply:
You can say the same for a hankerchief, but that doesn't really mean much.